clients/python-apiserver-client/README.md

```diff
@@ -0,0 +1,31 @@
+# Python client for KubeRay API server
+
+This Python client is currently only supporting Ray cluster management through usage of the `API server` Ray API. It implements all of the current functionality of the API server (and the operator) and provide pythonic APIs to the capabilities.
+
+The package supports well documented in the code Python objects that can be used to build and receive payloads for creation, listing and deleting of [template](https://ray-project.github.io/kuberay/components/apiserver/#compute-template) and Ray clusters.
```

Comment:
I'm not sure what "well documented in the code" is adding here. If you remove these words, does it make sense?